typescript/src/scss/main.scss

```diff
@@ -53,46 +52,41 @@
       }
     }
 
-
-    .grouped {
-      background-color: $low_opacity_gray;
-    }
-
-    /* Tooltip text */
     .tooltip{
       position: relative;
       padding: 1.115%;
+      .tooltiptext {
+        visibility: hidden;
+        width: 120px;
+        background-color: $dark_gray;
+        color: $white;
+        text-align: center;
+        padding: 5px 0;
+        border-radius: 6px;
+        position: absolute;
+        z-index: 1;
+        display: block;
+        bottom: 100%;
+      }
+      .tooltiptext::after {
+        content: " ";
+        position: absolute;
+        top: 100%; /* At the bottom of the tooltip */
+        left: 50%;
+        margin-left: -5px;
+        border-width: 5px;
+        border-style: solid;
+        border-color: $dark_gray transparent transparent transparent;
+      }
     }
 
-    .tooltip .tooltiptext {
-      visibility: hidden;
-      width: 120px;
-      background-color: $dark_gray;
-      color: $white;
-      text-align: center;
-      padding: 5px 0;
-      border-radius: 6px;
-      position: absolute;
-      z-index: 1;
-      display: block;
-      bottom: 100%;
-    }
-
-    /* Show the tooltip text when you mouse over the tooltip container */
     .tooltip:hover .tooltiptext {
       visibility: visible;
     }
-    .tooltip .tooltiptext::after {
-      content: " ";
-      position: absolute;
-      top: 100%; /* At the bottom of the tooltip */
-      left: 50%;
-      margin-left: -5px;
-      border-width: 5px;
-      border-style: solid;
-      border-color: $dark_gray transparent transparent transparent;
-    }
 
+    .grouped {
+      background-color: $low_opacity_gray;
+    }
```

Comment:
Could `.grouped` be moved inside `td`?

Reply:
<img width="840" alt="Screenshot 2021-11-29 at 13 23 56" src="https://user-images.githubusercontent.com/73976500/143875871-048e6e12-9cfe-4feb-a427-affb1e8ddb2b.png">
Doesn't work, gets rid of the shading

Reply:
OK, so either the selector isn't working or there must be another rule for the background colour that overrides this selector. If you use Chrome dev tools you should be able to see what's setting the background colour to work out what's going on - let me know if you want me to show you how.

Reply:
It works fine when moved tot body, I think the alternate row colouring overrides it when put in td.